### PR TITLE
Stopped rendering List & ListITem in `TableCellListItem`

### PR DESCRIPTION
**Pull request by ruchikabgosain**
This PR changes TableCellListItem to InheritMaterialUI across all files and folders for better understanding of the code. 
Also, adds `component=span` and `component=div` in taskcluster/ui/src/components/InheritMaterialUI/index.jsx file.

### CODE REVIEW

**Comment by Shubhamchinda** [comment]
> This PR changes TableCellListItem to InheritMaterialUI across all files and folders for better understanding of the code.
> Also, adds `component=span` and `component=div` in taskcluster/ui/src/components/InheritMaterialUI/index.jsx file.

Hey, please refer this to #627 and tag the author. That will help them to not miss the PR.

P.S: I refferred it.

File: ui/src/components/InheritMaterialUI/index.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ import ListItem from '@material-ui/core/ListItem';
 /**
  * A styled ListItem to be used when placed immediately after a TableCell.
  */
-export default class TableCellListItem extends Component {
+export default class InheritMaterialUI extends Component {
```
Comment (helfi92) [inline ui/src/components/InheritMaterialUI/index.jsx]: Let's rename to `TableCellItem`.

**Comment by ruchikabgosain** [comment]
Noted @Shubhamchinda! Thanks for your time :) 

@helfi92, is there anything more to be done?

File: ui/src/components/InheritMaterialUI/index.jsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ import ListItem from '@material-ui/core/ListItem';
 /**
  * A styled ListItem to be used when placed immediately after a TableCell.
  */
-export default class InheritMaterialUI extends Component {
+export default class TableCellItem extends Component {
```
Comment (helfi92) [inline ui/src/components/InheritMaterialUI/index.jsx]: We will need to change all instances of `InheritMaterialUI` to `TableCellItem`.

Comment (ruchikabgosain) [reply ui/src/components/InheritMaterialUI/index.jsx]: > done!

**Pull request closed without merging**